src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Main methods
+    operate(callback, value) {
+      callback(this, value);
+
+      return this;
+    },
```

Comment:
```suggestion
     operate: function(callback, num) {
      callback.call(this, num);

      return this;
    },
```

Reply:
[CODE KNOWLEDGE] - if you creating a method in the object, you don't need to use function keyword, use shortcut instead.
BAD EXAMPLE:

 methodName: function() {
 // your code
},
GOOD EXAMPLE:

 methodName() {
 // your code
},

Reply:
Thanks, with callback it works! Strange that it isn't in course